pkgs/development/libraries/qpdf/default.nix

```
@@ -11,20 +11,31 @@
 , pdfmixtool
 , pdfslicer
 , python3
+, testers
+, versionCheckHook
 }:
 
-stdenv.mkDerivation rec {
+stdenv.mkDerivation (finalAttrs: {
   pname = "qpdf";
   version = "11.9.1";
 
   src = fetchFromGitHub {
     owner = "qpdf";
     repo = "qpdf";
-    rev = "v${version}";
+    rev = "v${finalAttrs.version}";
     hash = "sha256-DhrOKjUPgNo61db8av0OTfM8mCNebQocQWtTWdt002s=";
   };
 
-  nativeBuildInputs = [ cmake perl ];
+  outputs = [
+    "bin"
+    "dev"
+    "doc"
+    "lib"
+    "man"
+    "out"
+  ];
+
+  nativeBuildInputs = [ cmake perl versionCheckHook ];
```

If [the docs for versionCheckHook](https://github.com/NixOS/nixpkgs/blob/7e7c39ea35c5cdd002cd4588b03a3fb9ece6fad9/doc/hooks/versionCheckHook.section.md?plain=1#L16-L19) are correct, then you should do:

```suggestion
  nativeBuildInputs = [
    cmake
    perl
  ];
  
  nativeInstallCheckInputs = [
    versionCheckHook
  ];
  doInstallCheck = true;

```

---

I remember reading this when you first reviewed but then I completely forgot 🤦🏻
Fixed!